django/dispatch/dispatcher.py

```diff
@@ -208,6 +209,8 @@ def send_robust(self, sender, **named):
             try:
                 response = receiver(signal=self, sender=sender, **named)
             except Exception as err:
+                logger = logging.getLogger("django.dispatch.dispatcher")
+                logger.exception(err)
```

Comment:
`logger.exception` does *not* take an exception instance, but a message ([docs](https://docs.python.org/3.9/library/logging.html#logging.Logger.exception)).

Using the exception will cause it to have `str()` cast on it, which is not useful in most cases.

I think we'd like to use a templated message like "{signal_name}.send_robust() received {exception_class_name} from {handler_name}".

When using `logger.exception`, there's no need to pass `err` in, as it's automatically captured since it's the 'currently handled exception'. It could be better to use `logger.error(msg, exc_info=err)` as this seems to be the pattern throughout the codebase - there are no pre-existing calls to `logger.exception`.

Reply:
we can use the templated message and we can get the values of _exception_class_name_ and _handler_name_ 
but how will we get the value of _signal_name_ as the name is just a variable?